modules/dnn/test/test_misc.cpp

```diff
@@ -78,6 +78,18 @@ TEST(readNet, Regression)
     EXPECT_FALSE(net.empty());
 }
 
+TEST(dump, Regression)
+{
+    Net net = readNet(findDataFile("dnn/squeezenet_v1.1.prototxt"),
+                      findDataFile("dnn/squeezenet_v1.1.caffemodel"));
```

Comment:
`.caffemodel`files are not in opencv_extra. So this test data is considered optional.
See #15052

Reply:
`dnn/squeezenet_v1.1.prototxt` is in opencv_extra. `false` is not needed there.
(issue is catched by `OPENCV_TEST_CHECK_OPTIONAL_DATA=1`)